src/dom.js

```diff
@@ -98,11 +98,19 @@ export const _DOM = {
       }
     }
   },
-  injectStyles(styles: string, destination?: Element, prepend?: boolean): Node {
+  injectStyles(styles: string, destination?: Element, prepend?: boolean, id?: string): Node {
     let node = document.createElement('style');
     node.innerHTML = styles;
     node.type = 'text/css';
 
+    if (id) {
+        let oldStyle = document.getElementById(id);
```

Comment:
A question from me. If we are finding a style with the provided ID, should we update that element instead of removing the old one?

Reply:
Again I will say what we think 😄 

Do these two approaches differ in terms of performance? If yes, cause we are not front-end experts, we vote for the fastest approach. and I speculate that updating would be faster owing to the fact that it does not contain any DOM removal or addition, am I right?

Reply:
That is correct. I just threw together a very, very basic test to get some numbers to make sure. It simply performs 10,000 operations of removing and inserting a new element and 10,000 operations of updating an element.

**Update existing element:** 197ms
**Insert new element:** 287ms

Code used for test:
```html
  <div id="container">
   <style id="mydiv">
     .bar {
       blue;
     }
    </style>
    <div class="bar">
      Foo
    </div>
  </div>
```

```javascript
let container = document.getElementById('container');

console.log('New test');
console.log('Update test started at: ' + new Date().getTime());

for (let i = 0; i < 10000; i++) {
  let div = document.getElementById('mydiv');
  
  div.innerHTML = '.bar { color: red; }';
}

console.log('Update test ended at: ' + new Date().getTime());

console.log('Insert test started at: ' + new Date().getTime());
  
for (let i = 0; i < 10000; i++) {
  let div = document.getElementById('mydiv');
  
  div.remove();
  
  let ele = document.createElement('style');

  ele.id = 'mydiv';
  ele.type = 'text/css';
  ele.innerHTML = '.bar { color: red; }';
  
  container.appendChild(ele);
}

console.log('Insert test ended at: ' + new Date().getTime());
```

Reply:
@ZHollingshead, I did it I am not a front-end developer so I hope it works!

Reply:
@ZHollingshead your results is what is expected to be  👍  , thanks for your time and effort 🙏